core-concepts/connecting-to-data-sources/connect-to-apis/authentication/README.md

```diff
@@ -0,0 +1,6 @@
+## Authentication
+
+In case the APIs that you wish to connect to are protected by an authentication mechanism, you can set up the requisite configuration at the data source level. Appsmith currently supports the following authentication mechanisms:
```

Comment:
@nidhi-nair I think there was a miscommunication here. We don't need a separate authentication file. The basic and oauth2 files can simply come under connect to apis